community/contributing/docs/github-changes.md

```diff
@@ -9,10 +9,10 @@ sidebar:
 
 # Open a pull request on GitHub
 
-1. Navigate to the [Spinnaker documentation repository](https://github.com/spinnaker/spinnaker.github.io). Note that the top navigation of Spinnaker.io corresponds to folder names. Use them as a reference to help you find the file you're looking for.
-2. On the page where you see the issue, select the pencil icon at the top right. You can also scroll to the bottom of the page and select **Edit this page**.
-3. Make your changes in the GitHub markdown editor.
-4. Below the editor, fill in the **Propose file change** form.
+1. Click the **Suggest an Edit** link in the upper right hand corner of the Spinnaker documentation page. This takes you to the file in GitHub.
+1. Select the pencil icon at the top right. You can also scroll to the bottom of the page and select **Edit this page**.
```

Comment:
Same thing here. Maybe "pencil icon at the top right" --> "**edit** button" or "**Edit this file** button".

Reply:
I followed the steps. I didn't see an "Edit this page" link or button anywhere, only the pencil icon. Removed the directional but left pencil icon.

Reply:
"Edit this file" is the hover text for the icon, so that's what someone using a sceen reader would hear. Of course, that confuses sighted users, who only see the pencil. :) No biggie for now, but I'll try to remember to look up best practices for this.

Reply:
Something like "Click the **Edit this file** (✏️) button.

Reply:
where did you get that pencil icon? Unfortunately I can't see the comment source.

Reply:
Added an image that points out the pencil icon.